app.js

```diff
@@ -1,90 +1,116 @@
 let fs = require('fs'),
     mysql = require('mysql'),
-    parse = require('csv-parse');
+    parse = require('csv-parse'),
+    filesToInsert,
+    isUserSpecified = false;
 
 const DB_CONFIG = require('./dbconfig.json');
 
 // function makes any unvaild csv's name valid
 function convertFileNameToTableName(file) {
     file = file.slice(0, -4).replace(/`/g, "``");
-
     return "`" + file + "`";
 }
 
-// Enumerate all files inside the "files" folder
-fs.readdir('./files/', 'utf8', (error, files) => {
-    if (error) {
-        console.error(error);
-        return;
+
+let args = process.argv;
+
+if (args[2] === "--files") {
```

Comment:
Also if I run the following command:
`node app.js --overwrite --files test.csv`
Then the program will simply use the files in the files folder, not the test.csv file that I specified

I think you should add test cases for both of these before fixing them